src/main/resources/fileTemplates/testMeTests/Parameterized Groovy, Spock & Mockito.groovy.ft

```diff
@@ -22,7 +22,7 @@ class ${CLASS_NAME}  extends Specification {
     }
 #end
 #foreach($method in $TESTED_CLASS.methods)
-#if($TestSubjectUtils.shouldBeTested($method))
+#if($hasMocks && $TestSubjectUtils.shouldBeTested($method))
```

Comment:
the additional condition `$hasMocks` - should be applied further down the line before `$MockitoMockBuilder.shouldStub` condition. test method generation should not be affected